lib/lidar_calculations.cpp

```diff
@@ -0,0 +1,67 @@
+#include <iostream>
+#include <vector>
+#include <ros/ros.h>
+#include "lidar_calculations.h"
+#include "lidar_point.h"
+#include <eigen3/Eigen/Dense>
+
+// Default constructor
+lidarCalculations::lidarCalculations() {}
```

Comment:
C++ automatically generates a constructor, so unless you want to do something specific in the constructor, you can remove this.